docs/IgnoreSuppressionConditions.md

```diff
@@ -102,7 +102,7 @@ A user would see the following output when attempting to synthesize an applicati
 
 `cdk-nag` exposes both a `SuppressionIgnoreAnd` class and a `SuppressionIgnoreOr` to help developers create more complicated conditions
 
-- `SuppressionIgnoreAnd`: Ignores the suppression if **ALL** of the given INagSuppressionIgnore return a non-empty message (logical and)
+- `SuppressionIgnoreAnd`: Ignores the suppression if **ALL** the given INagSuppressionIgnore return a non-empty message (logical and)
```

Comment:
This change isn't necessary. 'All of' is proper grammar

Reply:
Reverted this one and the one in Nagpacks.md